src/main/java/seedu/address/logic/commands/StatsCommand.java

```diff
@@ -12,43 +11,14 @@
 public class StatsCommand extends Command {
 
     public static final String COMMAND_WORD = "stats";
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Displays the number of GOOD, HARD & EASY flashcards ";
-
-    private final RatingContainsKeywordPredicate predicateGood = new RatingContainsKeywordPredicate("good");
-    private final RatingContainsKeywordPredicate predicateHard = new RatingContainsKeywordPredicate("hard");
-    private final RatingContainsKeywordPredicate predicateEasy = new RatingContainsKeywordPredicate("easy");
+    public static final String MESSAGE_USAGE = "Displays the number of GOOD, HARD & EASY FlashCards";
 
     ////////// Should return a CommandResult with the statistics ////////////////
```

Comment:
Good comment! I think it would be even better if you made this into a JavaDoc comment.